build.gradle

```diff
@@ -99,7 +99,7 @@ processResources.dependsOn writeBuildProperties
 build.dependsOn shadowJar, groovydoc, cloneContribFiles
 
 // FIXME: use ShadowJar (java -jar build/libs/web-api-skeleton-all.jar server configuration.yaml)
-task run(type: JavaExec, dependsOn: build) {
+task run(type: JavaExec) {
```

Comment:
If we decide we want to do this, we should change the skeleton. I'm ok with run not depending on build - we only do a run on our local machines for development. I just don't want some apis to depend on build, and others not.